packages/chart/package.json

```diff
@@ -42,19 +42,19 @@
   },
   "peerDependencies": {
     "@deephaven/log": "^0.3.1",
-    "react": "^16.x"
+    "react": "^17.x"
```

Comment:
Not caused by you, but we should probably be consistent with our peer dependencies format. In some places we do `^17.x`, in others we do `^17.0.0` ... I think I prefer the latter, but I leave that as an exercise for another time.